sites/docs/docs/guides/usql-migration-guide.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Universal SQL Migration Guide
+sidebar_position: 3
+---
+
+Universal SQL is Evidence's latest release, and fundamentally changes how Evidence queries data in order to bring support for three big new features:
+1. **Multiple Data Sources:** Combine data from multiple databases, flat files like CSVs, and even non-SQL data sources like Google Sheets - and query them all with SQL
+2. **Inputs and Filters:** Use input components to dynamically update your queries and charts based on user input. Filter data using parameterized queries rather than Javascript filter syntax
+3. **Adapter Plugins:** Create your own adapter to enable connecting to any data source
+
+This is a big release and contains several breaking changes. This guide will help you migrate your existing projects to USQL.
+
+
```

Comment:
Put a TLDR here about the migration script?

The following steps explain the changes to Evidence with Universal SQL.

If you are trying to migrate an existing project, jump straight to the migration script.